src/lib/metrics.ts

```diff
@@ -512,16 +858,6 @@ export default class MetricsMonitor {
                 legacyTokensActive.reset();
                 legacyTokensActive.set(deprecatedTokens.activeLegacyTokens);
 
-                if (maxEnvironmentStrategies) {
```

Comment:
This got migrated to: https://github.com/Unleash/unleash/pull/8450/files#diff-0610ce1595d1dcf150ca2efb2f0c5806d082f5bc6b8429a437970c7d60171cbdR181-R194 where both the gauge definition and refresh query are together